03-feedback.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Instructor Training
+subtitle: Feedback Strategy
+minutes: 180
+---
+> ## Learning Objectives {.objectives}
+>
+> * FIXME
+
+##Introduction
+
+In our discussion of our three-stage model of expertise, we concluded that most scientists don't need to be expert software developers; it's enough for them to be able to solve a recurring cast of coding problems in more-or-less familiar circumstances, which is the halmark of a competent practitioner. Teaching, however, is in some sense always a novel problem. Students will always come up with new ways to misunderstand things, and so the flexibility afforded by mastery in teaching is always valuable. How does one go about becoming a master instructor?
+
+The popular ten thousand hours number is certainly a myth. Many people have driven for more than ten thousand hours, and are little closer to mastery than after their driver's exam. Something more than mere repetition is required.
+
+Positing additional linkages between ideas is necessary but also not sufficient; there needs to be some mechanism to scrutinize and validate those new links and the strategies that come out of them.
+
+Musicians, athletes, actors and politicians have been doing it for years - the key is simple feedback. By having a teacher or even a colleague observe and give constructive criticism that is specific and actionable, we can iterate on our practice and get that invaluable second perspective on whether what we're doing is working, and where it's falling short. In time, with enough criticism, we can begin to usefully criticize our own practice, and this self-reflective criticism greatly accelerates the process of gaining mastery. But to begin with, getting feedback from our colleagues starts the process.
+
+##Anatomy of Feedback
+
+The practice of giving feedback can be incredibly powerful in its ability to expose things we couldn't perceive in ourselves in the intensity of the stage, but it can also be a very fraught process. Criticism can challenge our self image, and chafe on our insecurities and doubts; receiving criticism can feel like a place of powerlessness. If we are to successfully critique each other, we must find a way to speak our minds in a way each other can hear - how do we criticize each other in a way designed to empower and not to tear down? Giving good criticism consists of three cardinal rules:
```

Comment:
Again, this seems a bit strong.  Is there research backing up these three rules?  I imagine there is a good deal of research on criticism, but I'm not familiar with any of it.  In the absense of a more evidence-based argument, I'd prefer wording like: “Here are a few techniques that may help you give constructive criticism:”